lib/index.js

```diff
@@ -60,9 +60,9 @@ function SpmAgent (spmUrl) {
     // logger.warn ('sendFailed event ' + JSON.stringify (counters))
     this.emit('stats', counters)
   }.bind(this))
-  this.influxSender.on('error', function (e) {
+  this.spmSender.on('error', function (e) {
```

Comment:
Should we try to get away from "spm"?

Reply:
This code is 5yrs old. Shall we completely refactor it? The plan is to remove spmSender as soon we have a replacement for mongodb, httpd and nginx agent. Those use the old receiver interface.

Reply:
We will keep the variable names for now. As I mentioned the plan is to remove the spmSender module. We had to keep it for backward compatibility with nginx, httpd, mongodb agent. We can not change all agents at once.

Reply:
👍